### DISCUSSION TITLE
User accounts

### DISCUSSION THREAD

**Discussion by spurnasai** [WeblateOrg/weblate#12178] [Q&A]
Is there is any built-in feature in the weblate so that it sends a notification to the admin when a new user logs in ?

**Comment by nijel** [comment]
No, there is currently no such a feature.

**Reply by spurnasai** [reply]
So I want to use my own CLA(Contributor License Agreement) ,where should I specify that link so if the users want to contribute any thing they has to sign the CLA.  Is that possible ,If yes How can we do that ?

**Reply by nijel** [reply]
There is built-in CLA per-project in Weblate: https://docs.weblate.org/en/latest/admin/projects.html#contributor-agreement

If you want something site-wide, use legal module: https://docs.weblate.org/en/latest/admin/optionals.html#legal

**Reply by spurnasai** [reply]
I have gone through the contributor-agreement but there are no steps as such how to do CLA per project

**Reply by spurnasai** [reply]
This is our CLA link : https://oss-cla.sangoma.com/freepbx/sng_freepbx_debian_install 
1) How can we integrate  this to the weblate project ? and 
2) If the CLA is enabled is there  any module as such so that it will show the list of users who are pending for signing the CLA and who has already signed the CLA  and waiting for project access ?

**Reply by nijel** [reply]
Users can't contribute util they agree with the contributor agreement.

**Reply by spurnasai** [reply]
I am not finding any option regarding the contributor license agreement in the UI or in the settings_docker.py How to enable that one ?

**Reply by nijel** [reply]
It's in the component configuration (sorry for confusion, it's not project level, but component level).

**Reply by spurnasai** [reply]
While trying to save the Contributor agreement ,it is asking for the Translation license How to enable that one ?

**Reply by nijel** [reply]
What is the actual error you're  getting?
